tsconfig.json

```diff
@@ -12,7 +12,13 @@
     "jsx": "react",
     "resolveJsonModule": true,
     "incremental": true,
-    "strictNullChecks": true
+    "strictNullChecks": true,
+    "skipLibCheck": true
```

Comment:
I assume this will be the controversial part of this PR, so I'm calling it right now. Have I misunderstood how this flag works, and does it make our codebase worse?

Reply:
https://twitter.com/orta/status/1260583041740812288
![image](https://user-images.githubusercontent.com/18808/151771681-dcef3fae-0324-4516-b414-6be37972d79e.png)